scheduler/scheduler.cpp

```diff
@@ -105,6 +105,7 @@ static list<CompileServer *> css;
 static list<CompileServer *> monitors;
 static list<CompileServer *> controls;
 static list<string> block_css;
+static list<CompileServer *> css_temp_hold;
```

Comment:
I'm uncomfortable with two lists that we need to maintain: it seems too likely that sometime in the future someone will change something and forget to check both.  I think we should create one class that maintains css, css_temp_hold and css_all. That way there is one source of the information we need and we can just ask for different iterators on it.